Install.md

```diff
@@ -21,19 +21,45 @@ Starting with version 1.1.0, the GCM is capable of configuring Git automatically
 1. Copy the `${project.artifactId}-${project.version}.jar` file somewhere safe and stable, such as `~/${project.artifactId}/`.
 2. Launch the JAR in `install` mode, which will check its requirements and then update the "global" Git configuration file (the one in your home folder):
 
-```
-java -jar /home/example/${project.artifactId}/${project.artifactId}-${project.version}.jar install
-```
+    ```
+    java -jar /home/example/${project.artifactId}/${project.artifactId}-${project.version}.jar install
+    ```

```

Comment:
It looks like you've discovered how to get Markdown to display code blocks without interfering with list item numbering by indenting them?